Foundation/NSCharacterSet.swift

```diff
@@ -50,13 +50,18 @@ open class NSCharacterSet : NSObject, NSCopying, NSMutableCopying, NSCoding {
     }
     
     open override func isEqual(_ value: Any?) -> Bool {
+        guard let runtimeClass = _CFRuntimeGetClassWithTypeID(CFCharacterSetGetTypeID()) else {
```

Comment:
@phausler Could you please explain a bit more? Isn't this what `CFEqual` does under the hood?

I'm open for suggestions.